### PR TITLE
refactor(egen) : refactored notebook according to template guidelines nd added missed modules and libraries

### PR DESCRIPTION
**Pull request by Jayakrishna2801**
This PR includes
1.replaced variable REGION to LOCATION
2.imported required modules
3.added missed libraries in installation section
4.added colab enterprise link
5.added Colab authentication and kernal restart steps.
6.refactored notebook according to template guidelines

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/workbench/sentiment_analysis/Sentiment_Analysis.ipynb [hunk 1]
```diff
@@ -103,10 +106,10 @@
...
         "This tutorial uses the following Google Cloud ML services:\n",
         "\n",
-        "- `Vertex AI Dataset`\n",
-        "- `Vertex AI Model`\n",
-        "- `AutoML training`\n",
-        "- `Vertex AI Prediction`\n",
+        "- Vertex AI dataset\n",
+        "- Vertex AI model\n",
+        "- AutoML training\n",
+        "- Vertex AI prediction\n",
```
Comment (kittyabs) [inline notebooks/official/workbench/sentiment_analysis/Sentiment_Analysis.ipynb:112]: - change: Vertex AI prediction
- to: Vertex AI Prediction

**Review by kittyabs — COMMENTED** [review]
some edits. Wondering why this one was updated since it references: 
https://cloud.google.com/vertex-ai/docs/training-overview#text_data, which is being deprecated

**Comment by Jayakrishna2801** [comment]
> some edits. Wondering why this one was updated since it references: https://cloud.google.com/vertex-ai/docs/training-overview#text_data, which is being deprecated

> mistake, replaced with original reference : https://cloud.google.com/vertex-ai/docs/datasets/prepare-text#sentiment-analysis

**Review by katiemn — APPROVED** [review]

**Pull request merged**